nimble/controller/src/ble_ll_hci.c

```diff
@@ -1075,54 +999,37 @@ ble_ll_hci_le_cmd_proc(uint8_t *cmdbuf, uint16_t ocf, uint8_t *rsplen,
 #endif
 #if (BLE_LL_BT5_PHY_SUPPORTED == 1)
     case BLE_HCI_OCF_LE_RD_PHY:
-        if (len == BLE_HCI_LE_RD_PHY_LEN) {
-            rc = ble_ll_conn_hci_le_rd_phy(cmdbuf, rspbuf, rsplen);
-        }
+        rc = ble_ll_conn_hci_le_rd_phy(cmdbuf, len, rspbuf, rsplen);
         break;
     case BLE_HCI_OCF_LE_SET_DEFAULT_PHY:
-        if (len == BLE_HCI_LE_SET_DEFAULT_PHY_LEN) {
-            rc = ble_ll_hci_le_set_def_phy(cmdbuf);
-        }
+        rc = ble_ll_hci_le_set_def_phy(cmdbuf, len);
         break;
     case BLE_HCI_OCF_LE_SET_PHY:
-        if (len == BLE_HCI_LE_SET_PHY_LEN) {
-            rc = ble_ll_conn_hci_le_set_phy(cmdbuf);
-        }
+        rc = ble_ll_conn_hci_le_set_phy(cmdbuf, len);
         break;
 #endif
 #if MYNEWT_VAL(BLE_LL_DIRECT_TEST_MODE)
-    case BLE_HCI_OCF_LE_ENH_RX_TEST:
-        if (len == BLE_HCI_LE_ENH_RX_TEST_LEN) {
-            rc = ble_ll_dtm_rx_test(cmdbuf, true);
-        }
+    case BLE_HCI_OCF_LE_RX_TEST_V2:
```

Comment:
oh, you are changing also names in the same time according to v5.1 - that should be mentioned in commit message